CSS/styles.css

```diff
@@ -0,0 +1,182 @@
+* {
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
+
+nav {
+  width: 100%;
+  height: 50px;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 30px;
+  border: 2px solid #000;
+
+  /* background-color: aliceblue; */
+}
+
+.list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  column-gap: 40px;
+  position: relative;
+}
+
+.books {
+  padding-top: 10px;
+  font-size: 1.5rem;
+  font-weight: 700;
+}
+
+.list li:not(:last-child)::after {
+  border: 1px solid #000;
+  border-width: 1px 1px 0 0;
+  position: absolute;
+  right: -25px;
+  top: 0;
+  height: 100%;
+}
+
+.date {
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 30px;
+  padding-top: 10px;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  padding-bottom: 50px;
+}
+
+.main-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  margin-top: 10rem;
+}
+
+.heading {
+  margin: 0 auto;
+  text-align: center;
+  font-size: 2.5rem;
+}
+
+/* .all-books {
+  margin-top: 2rem;
+  width: 50vw;
+  border: 1px solid #000;
+} */
```

Comment:
Kindly take away every commented block of code in your codebase as it makes it messy and gives it a less professional look👍🏼.